### ISSUE TITLE
Fetch some of the last posts of a newly visited remote profile

### ISSUE THREAD

**Issue by xplosionmind** [pixelfed/pixelfed#3115]
The discussion in #2933 made me think that it would be very nice if, when a user searches for a remote profile that has never been contacted yet of an instance that is not federated, Pixelfed could request to the other instance some of the last posts of that user.

As of right now, visiting the profile page of a remote user from your own instance for the first time shows an empty page. Only the bio is visible, not even the profile picture is displayed!

**Comment by inthreedee** [comment]
This was a big source of confusion for me after setting up my instance.  I had followed myself back and forth with my pixelfed.social account for testing, so the connection was established.  When I posted on my instance, however, I wasn't seeing anything show up on my pixelfed.social account.  It wasn't until I put up a test post on pixelfed.social that finally things seemed to start working.

> when a user searches for a remote profile that has never been contacted yet of an instance that is not federated

Or even when the instance *is* federated but the accounts aren't following one another yet.  For example, my instance is now federated with pixelfed.social, but if I search for `https://pixelfed.social/dansup` from my instance, I still see an empty profile.  It would seem that I would have to follow that profile and then wait for a new photo to be posted before anything starts working.
